Xamarin.Forms.Core/InputView.cs

```diff
@@ -5,6 +5,14 @@ public class InputView : View
 		public static readonly BindableProperty KeyboardProperty = BindableProperty.Create("Keyboard", typeof(Keyboard), typeof(InputView), Keyboard.Default,
 			coerceValue: (o, v) => (Keyboard)v ?? Keyboard.Default);
 
+        public static readonly BindableProperty MaxLengthProperty = BindableProperty.Create("MaxLength", typeof(int), typeof(int), int.MaxValue);
```

Comment:
I'd use `-1` as default value

Reply:
use `nameof(MaxLength)`

Reply:
we usually use tabs for indentation

Reply:
In the #1663 we sort of decided to go with int.MaxValue. What would be the benefit of using -1?
Fixed nameof and tabs.

Reply:
I'll discuss the defaultValue with @hartez. both are fine, both make sense.